modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc

```diff
@@ -10,6 +10,58 @@ New features are added to the language continuously, and occasionally, some feat
 This section lists all of the features that have been removed, deprecated, added, or extended in different Cypher versions.
 Replacement syntax for deprecated and removed features are also indicated.
 
+
+[[cypher-deprecations-additions-removals-5.8]]
+== Version 5.8
+
+=== New features 
+
+[cols="2", options="header"]
+|===
+| Feature
+| Details
+
+a| 
+label:functionality[]
+label:new[]
+
+New operator: `AssertSameRelationship`
+
+a| 
+The `AssertSameRelationship` operator is used to ensure that no relationship property uniqueness constraints are violated in the slotted and interpreted runtime.
+More information can be found xref::execution-plans/operators.adoc#query-plan-assert-same-relationship[here].
+
+|===
+
+[[cypher-deprecations-additions-removals-5.8]]
+== Version 5.8
```

Comment:
Possibly these two lines need to be deleted

Reply:
cool, thanks - good spot